src/modules/EventList/index.tsx

```diff
@@ -433,6 +438,67 @@ export const EventList = (_props: EventListProps): ReactElement => {
     useAutoTrack('Click', 'Event Filter By', { filterBy: state.filterByTypes }, [state.filterByTypes]);
     useAutoTrack('Submit', 'Event Search', { searchTerm: state.searchTerm }, [state.searchTerm], !state.searchTerm);
 
+    // Check AlertList against current time
+    // and fire events
+    const { alertList, removeDateKey } = useAlertList();
+    useEffect(() => {
+        if (eventsQuery.status === 'success') {
+            const dateKeys = alertList.dateKeys?.map((dk) => new Date(dk).getTime());
+            const events = eventsQuery.data.events;
+
+            const checkAlerts = setInterval(() => {
+                const currentTime = new Date().getTime();
+                const tickerList: string[] = [];
+                let fireAlert = false;
+                let eventIds: string[] = [];
+                dateKeys.forEach((dk, index) => {
+                    if (currentTime > dk) {
```

Comment:
What happens if the alert was for something a while ago? Does it still fire?

Reply:
I've changed this logic to only fire if the event started within the last two hours